.github/workflows/push-trigger.yml

```diff
@@ -51,8 +51,56 @@ jobs:
     if: "${{ github.event_name != 'pull_request' }}"
     uses: mosip/kattu/.github/workflows/npm-sonar-analysis.yml@master
     with:
-      SERVICE_LOCATION: ./resident-ui
+      SERVICE_LOCATION: resident-ui
```

Comment:
Your branch is not up to date 

Please update it and resolve the merge conflicts